azurerm/internal/services/web/resource_arm_function_app.go

```diff
@@ -97,7 +97,7 @@ func resourceArmFunctionApp() *schema.Resource {
 			},
 
 			"connection_string": {
-				Type:     schema.TypeList,
+				Type:     schema.TypeSet,
```

Comment:
in addition to this we also need to update [the expand function](https://github.com/terraform-providers/terraform-provider-azurerm/pull/5319/files#diff-23333db7a329c446abb71d2877f1bcf1R803) to use: `d.Get("foo").(*schema.Set).([]interface{}` rather than `d.Get("foo").([]interface{})` since this is now a Set rather than a List